build.gradle

```diff
@@ -483,7 +483,7 @@ bootWithCCD {
     //environment 'SERVICEBUS_FAKE', 'true'
 }
 
-pmdMain.maxFailures = 422
+pmdMain.maxFailures = 324
 pmdTest.maxFailures = 227
```

Comment:
I see there were changes in tests so this number may have come down too

Reply:
No unfortunately, I've ran 'gradle pmdTest' in both master & my branch and the number is still the same '227'